packages/firebase_data_connect/firebase_data_connect/example/dataconnect/src/main/java/com/myapplication/AddMovieMutation.kt

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2024, the Chromium project authors.  Please see the AUTHORS file
+ * for details. All rights reserved. Use of this source code is governed by a
+ * BSD-style license that can be found in the LICENSE file.
+ */
+
+
+@file:Suppress(
```

Comment:
Do we need this Kotlin file? If I understand properly we only need the Dart generated files